.appveyor.yml

```diff
@@ -0,0 +1,171 @@
+  APPVEYOR_CACHE_ENTRY_ZIP_ARGS: -t7z -snl -mtc -mx=1
+
+cache:
+  # Python packages installed with `pip --user` and Pip cache.
+  - $(APPDATA)\Python
+  - $(LOCALAPPDATA)\pip
+  # Rust stuff.
+  - $(CARGO_HOME)
+  - $(RUSTUP_HOME)
+  # Cache the third_party submodule to preserve binaries downloaded by setup.py,
+  # and to make incremental builds work.
+  - $(APPVEYOR_BUILD_FOLDER)\.git\modules\third_party
+  - $(APPVEYOR_BUILD_FOLDER)\third_party
+  # Build incrementally.
+  - $(DENO_BUILD_PATH)
+
+init:
+  # TODO: enable after this lands in master.
+  # Do no save the build cache for feature branches. Once we have multiple
+  # "official" branches, use a build matrix so each branch has it's own cache.
+  # - ps: |
+  #     if ($env:APPVEYOR_REPO_BRANCH -ne "master") {
+  #       $env:APPVEYOR_CACHE_SKIP_SAVE = "true"
+  #     }
+
+  # Make git check out symlinks (not placeholder text files).
+  - git config --global core.symlinks true
+
+install:
+  # `Exec` runs a regular executable. It looks at the process' exit code,
+  # rather than its stderr output, to tell if a command has failed
+  - ps: |
+      function Exec([ScriptBlock] $Command, [Switch] $NoNewLines) {
+        "$Command".TrimStart(" &") | Write-Host              # Echo command.
+        & $Command 2>&1 | Write-Host -NoNewLine:$NoNewLines  # Execute command.
+        if ($NoNewLine) { Write-Host }                       # Write newline.
+        if ($LastExitCode -ne 0) { throw "Failure. Exit code: $LastExitCode" }
+      }
+
+  # Clone the third_party submodule.
+  - ps: |
+      try {
+        Exec { & git submodule update --init --force --depth 1 }
+
+      } catch {
+        # Git will fail if the `third_party` directory was restored from cache,
+        # but the `.git/modules` directory wasn't. Rebuild it from scratch.
+        Remove-Item -Path $env:DENO_THIRD_PARTY_PATH `
+                    -Recurse -Force -ErrorAction Ignore
+        Exec -NoNewLines { & git submodule update --init --force --depth 1 }
+      }
+
+  # Prune and pack git objects. Thus when we upload `.git/modules/` to
+  # the Appveyor cache, it'll include only objects that were actually needed.
+  - ps: |
+      if ($env:APPVEYOR_CACHE_SKIP_SAVE -ne "true") {
+        Push-Location $env:DENO_THIRD_PARTY_PATH
+        Exec { & git repack -d -l }
+        Pop-Location
+      }
+
+  # Install a recent Node.js version.
+  - ps: Install-Product node 10 x64
+
+  # Make sure the right Python version is in PATH, and others are not.
+  - ps: |
+      # Remove the wrong Python version(s) from PATH.
+      $p = $env:PATH -split ";" | Where-Object {
+             -not (Test-Path -Path "$_\python.exe") -and
+             -not (Test-Path -Path "$_\pip.exe")
+           }
+      # Add binary dir for `pip --user` packages.
+      $p += "$env:APPDATA\Python\Scripts"
+      # Add python27-x64.
+      $p += "c:\Python27-x64"
+      $p += "c:\Python27-x64\Scripts"
+      $env:PATH = $p -join ";"
```

Comment:
This is painful. Is this because we don't support py3? if so please add an issue for py3 support and link it here.

Reply:
The default is python2 (x86) but there was some issue with google's virtualenv.